samples/wifi/promiscuous/sysbuild.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+SB_CONFIG_WIFI_NRF700X=y
```

Comment:
This won't work, we need a RAW mode supported patch.

Reply:
Please test with sysbuild.

Reply:
> This won't work, we need a RAW mode supported patch.

Added.

Reply:
> Please test with sysbuild.

Tested and is working as expected.